examples/tools/cmake/cmake_toolchain/local_flow_cmake_presets.rst

```diff
@@ -0,0 +1,84 @@
+CMakeToolchain using CMakePresets
+=================================
+
+.. note::
+
+    You can find the following example in this repository
+
+    .. code-block:: bash
+
+        $ git clone https://github.com/conan-io/examples2.git
+        $ cd examples2/examples/tools/cmake/cmake_toolchain/local_flow_cmake_presets
+
+
+One of the advantages of using Conan toolchains is that they can help to achieve the same build
+with local development flows, as if the package is created in the cache.
```

Comment:
I would add a more detailed introduction, explaining for example what's the advantage of using `CMakePresets, what IDE's support this, etc.